game/theater/controlpoint.py

```diff
@@ -757,17 +812,22 @@ def aircraft_retreat_destination(
         self, squadron: Squadron
     ) -> Optional[ControlPoint]:
         closest = ObjectiveDistanceCache.get_closest_airfields(self)
-        max_retreat_distance = squadron.aircraft.max_mission_range
+        # Multiply the max mission range by two when evaluating retreats,
+        # since you only need to fly one way in that case
+        max_retreat_distance = squadron.aircraft.max_mission_range * 2
```

Comment:
Add "Improved squadron retreat logic" to changelog?

Reply:
Added.